web-server/v0.4/components/Comparison/CompareIterations.js

```diff
@@ -0,0 +1,534 @@
+              if (Object.keys(args[responseCount].data[iterationTypes[iterationTest]]).includes(primaryMetric)) {
+                var hosts = args[responseCount].data[iterationTypes[iterationTest]][primaryMetric];
+                for (var host in hosts) {
+                  if (hosts[host].client_hostname == "all") {
+                    for (var item in hosts[host].timeseries) {
+                      hosts[host].timeseries[item] = this.renameProp('date', 'x', hosts[host].timeseries[item])
+                      hosts[host].timeseries[item] = this.renameProp('value', 'y' + (parseInt(iteration) + 1), hosts[host].timeseries[item])
+                    }
+                    timeseriesLabels['y' + (parseInt(iteration) + 1)] = clusteredIterations[primaryMetric][cluster][iteration].result_name + '-' + clusteredIterations[primaryMetric][cluster][iteration].iteration_name;
+                    iterationTimeseriesData = _.merge(iterationTimeseriesData, hosts[host].timeseries);
+                    iterationNames.push(clusteredIterations[primaryMetric][cluster][iteration].result_name + '-' + clusteredIterations[primaryMetric][cluster][iteration].iteration_name);
+                    responseCount++;
+                  }
+                }
+              }
+            }
+          }
+          timeseriesData[primaryMetric][cluster].push({iterationNames: iterationNames, data: iterationTimeseriesData, labels: timeseriesLabels});
+        }
+      }
+      for (var primaryMetric in Object.keys(timeseriesData)) {
+        timeseriesDropdownSelected[Object.keys(timeseriesData)[primaryMetric]] = 0;
+        timeseriesDropdown[Object.keys(timeseriesData)[primaryMetric]] = Object.keys(timeseriesData[Object.keys(timeseriesData)[primaryMetric]]);
+      }
+      this.setState({timeseriesData: timeseriesData});
+      this.setState({timeseriesDropdownSelected: timeseriesDropdownSelected});
+      this.setState({timeseriesDropdown: timeseriesDropdown});
+    });
+  }
+
+  clusterDropdownChange = (value, primaryMetric) => {
+    var { timeseriesDropdownSelected } = this.state;
+    timeseriesDropdownSelected[primaryMetric] = value;
+    this.setState({timeseriesDropdownSelected: timeseriesDropdownSelected});
+  }
+
+  componentDidMount() {
+    this.setState({loading: true});
+    const { configCategories } = this.props;
+    this.generateIterationClusters(configCategories);
+  }
+
+  printSummary = () => {
+    console.log("clicked")
+    window.print()
+  }
+
+  iframe(controllerName, resultName, iterationNumber, iterationName, closestSample) {
+    return {
+      __html: '<iframe style="overflow: hidden; overflow-y: hidden; height: 650px; border: none; margin: 0; padding: 0; scrolling: none" src="http://pbench.perf.lab.eng.bos.redhat.com/results/' + controllerName + '/' + resultName + '/' + iterationNumber + '-' + iterationName + '/' + 'sample' + closestSample + '/uperf.html" width="100%" height="1000"></iframe>'
```

Comment:
Can we find a way to remove this hard-coded `http://pbench.perf.lab.eng.bos.redhat.com` URL prefix?